metis-schema/src/main/resources/schema_xsds/EDM-EXTERNAL-MAIN.xsd

```diff
@@ -53,7 +53,7 @@
           PORT: generic (no port) | fi | il | nl
           -->
           <pattern
-            value="^http[:]//creativecommons[.]org/licenses/(by|by-sa|by-nd|by-nc|by-nc-sa|by-nc-nd|by-nd-nc)/1[.]0/(fi|il|nl)?$"/>
+            value="^http[:]//creativecommons[.]org/licenses/(by|by-sa|by-nd|by-nc|by-nc-sa|by-nc-nd|by-nd-nc)/1[.]0/(fi/|il/|nl/)?$"/>
```

Comment:
Why not take the / out of the individual cases?
```
^http[:]//creativecommons[.]org/licenses/(by|by-sa|by-nd|by-nc|by-nc-sa|by-nc-nd|by-nd-nc)/1[.]0/((fi|il|nl)/)?$
```
The same for the other changes.